frontend/src/app/services/game.ts

```diff
@@ -58,59 +70,109 @@ export class Game {
+    addMitigation('events100', 0.34, [0.12, 0.52], 20 * cs(), 10 * ss(), 'Akce 100 lidí');
+    addMitigation('events10', 0.42, [0.17, 0.60], 40 * cs(), 20 * ss(), 'Akce 10 lidí');
+    addMitigation('businessesSome', 0.18, [-0.08, 0.40], 20 * cs(), 15 * ss(), 'Rizikové služby');
+    addMitigation('businessesMost', 0.27, [-0.03, 0.49], 40 * cs(), 30 * ss(), 'Většina služeb');
+    addMitigation('universities', 0.19, [0.05, 0.33], 10 * cs(), 10 * ss(), 'Univerzity', {isSchool: true});
+    addMitigation('schools', 0.38, [0.16, 0.54], 50 * cs(), 40 * ss(), 'Všechny školy', {isSchool: true});
+    addMitigation('stayHome', 0.13, [-0.05, 0.31], 50 * cs(), 30 * ss(), 'Zůstat doma');
+    addMitigation('borders', 0.00, [0.00, 0.00], 10 * cs(), 10 * ss(), 'Zavřít hranice', {isBorders: true});
+
+/*
+    The mitigation parameters as above, just formatted more nicely
+
+    // TODO Find reference?
+    // maybe? https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)31142-9/fulltext cites 14.3% for face masks?
+    addMitigation('rrr',            0.14, [0.10, 0.18],  5 * cs(),  2 * ss(), 'Ruce, Roušky, Rozestupy'); // Not from the paper
+
+    // https://science.sciencemag.org/content/early/2020/12/15/science.abd9338
+    addMitigation('events1000',     0.23,  [0.00, 0.40], 10 * cs(),  5 * ss(), 'Akce 1000 lidí');
+    addMitigation('events100',      0.34,  [0.12, 0.52], 20 * cs(), 10 * ss(), 'Akce 100 lidí');
+    addMitigation('events10',       0.42,  [0.17, 0.60], 40 * cs(), 20 * ss(), 'Akce 10 lidí');
+    addMitigation('businessesSome', 0.18, [-0.08, 0.40], 20 * cs(), 15 * ss(), 'Rizikové služby');
+    addMitigation('businessesMost', 0.27, [-0.03, 0.49], 40 * cs(), 30 * ss(), 'Většina služeb');
+    addMitigation('universities',   0.19,  [0.05, 0.33], 10 * cs(), 10 * ss(), 'Univerzity', {isSchool: true});  // 1/2 of the effect of schools (the paper couldn't robustly predict effect of universities alone)
+    addMitigation('schools',        0.38,  [0.16, 0.54], 50 * cs(), 40 * ss(), 'Všechny školy', {isSchool: true});
+    addMitigation('stayHome',       0.13, [-0.05, 0.31], 50 * cs(), 30 * ss(), 'Zůstat doma');       // Marginal effect of lockdown
+
+    // Controls drift across borders
+    addMitigation('borders',        0.00, [0.00, 0.00], 10 * cs(), 10 * ss(), 'Zavřít hranice', {isBorders: true});    // Not from the paper
+*/
+
+    // effectivityConfidence 2sigma confidence interval (can be asymmetric)
+    // isSchool mitigations are effective during school holidays "for free"
+    // isBorders controls epidemic drift across borders
+    function addMitigation(id: string, effectivity: number, effectivityConfidence: number[], costMPerDay: number, stabilityCost: number, label: string, flags?: any) {
       mitigations.push({
         id,
         label,
-        effectivity,
-        cost,
+        effectivity: effectivity + ens() * (effectivityConfidence[1] - effectivityConfidence[0]) / 4,
+        cost: costMPerDay,
```

Comment:
back and forth? let's minimize the changes, attributes do need explanations, you can document the method via jsdoc or the interface